app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,38 @@
+class StatsController < ApplicationController
+  def index
+    @player_week_stats = Player
+      .joins(:player_week_stats)
+      .select('player_week_stats.*, players.name')
+      .where(position: position)
+      .joins('INNER JOIN season_weeks ON season_weeks.id = player_week_stats.season_week_id')
+      .where('season_weeks.season = ? AND season_weeks.week = ?', season, week)
```

Comment:
@jmejia We know that this logic shouldn't live in the controller, but we don't believe that it should live in the Player model or PlayerWeekStat model either. Right now we plan on making this into a PORO, thoughts?